### PR TITLE
added .travis.yml

### PR DESCRIPTION
**Pull request by shreyagupta30**
Hello @leoank 
I have created the travis.yml file. Here is my work process

1. Used dist as Xenial to fast up the process
2. Defined entire matrix group as ```edge```
3. Now, all future segments of the entire codebase can be added in matrix by just defining it in this manner ```- language: ....```

### CODE REVIEW

File: intermine_boot/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+dist: xenial   #Use light Xenial distro
```
Comment (leoank) [inline intermine_boot/.travis.yml:1]: Try adding support for running test on MacOS and Windows

**Review by leoank — COMMENTED** [review]
Rest of it looks good.

File: intermine_boot/.travis.yml (hunk 1 above)
Comment (shreyagupta30) [reply intermine_boot/.travis.yml:1]: Okay sir!

**Comment by leoank** [comment]
closing this in favor of #39

**Pull request closed without merging**